static/src/javascripts/bootstraps/commercial.js

```diff
@@ -36,7 +38,8 @@ define([
         ['cm-articleBodyAdverts', articleBodyAdverts.init],
         ['cm-sliceAdverts', sliceAdverts.init],
         ['cm-frontCommercialComponents', frontCommercialComponents.init],
-        ['cm-commercialAudit', commercialAudit.init]
+        ['cm-commercialAudit', commercialAudit.init],
+        ['cm-hostedVideo', hostedVideo.init]
```

Comment:
Task for the future next generation of hosted content is to include only hosted scripts as we don't need most of the commercial stuff there.